Cue/app/actions/library.js

```diff
@@ -129,12 +130,40 @@ async function addLibrary(uuid: string): PromiseAction {
     }
   }
 
-  let deck = await LibraryApi.fetchDeck(uuid);
+  let addedDeck = await LibraryApi.fetchDeck(uuid);
 
   return {
     type: 'DECK_ADDED_TO_LIBRARY',
-    deck,
+    addedDeck,
   };
 }
 
-module.exports = { loadLibrary, createDeck, deleteDeck, editDeck, recordShareCode, syncDeck, syncLibrary, addLibrary, resolveConflict };
+function clearInaccessibleDecks(): Action {
+  return {
+    type: 'CLEAR_INACCESSIBLE_DECKS'
+  }
+}
+
+function copyDeck(localDeck: Deck) : Action {
+  let newCards = localDeck.cards.map(card => {
+    return {...card, action: 'add', uuid: undefined}
```

Comment:
Should `uuid` be a new UUID (to match [the change in 153](https://github.com/ClaytonPassmore/ProjectOrange/pull/153/files#diff-8ea7c9b29416c07c38ba89c50acc7f46R288) to fix #133)?